CONTRIBUTING.md

```diff
@@ -88,6 +88,45 @@ juju deploy ./superset-k8s_ubuntu-22.04-amd64.charm --resource superset-image=ap
 # Check deployment was successful:
 juju status
 ```
+
+### Authentication
+To validate Google Oauth authentication:
+```
+# Port forward the web server
+kubectl port-forward pod/superset-k8s-0 8088:8088 -n superset-k8s
+
+# Edit /etc/hosts
+sudo vim /etc/hosts
```

Comment:
Since this is the `CONTRIBUTING.md`, I assume this is just for charm developers. Even then, I do not think this is a nice thing to ask. I guess there is not a better way of doing this currently but can we leave a TODO or something somewhere to be on the lookout for improvements?

Reply:
Yes just charm developers. I can simplify this a bit as you can use localhost in the google redirect_uri. I was trying to validate here that the authorisation was working for a domain that could then be used in production. But I can simply request that localhost is updated to the host domain.